src/control/cmd/dmg/pretty/system.go

```diff
@@ -21,8 +23,17 @@ import (
 	"github.com/daos-stack/daos/src/control/system"
 )
 
+const randomStringLen = 5
+
+func randomString() string {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Could it be possible to define the seed in a way that it could be easily reproductible: setting the seed at the initialization of the command and be able to set its value.  Could be helpful for maintenance and debugging tasks.